frontend/src/components/dashboard/aliases/BlockLevelSlider.test.tsx

```diff
@@ -195,19 +197,24 @@ describe("BlockLevelSlider", () => {
 
     await user.click(ghostButton);
 
+    const tooltip = await screen.findByRole("dialog");
+
     expect(
-      screen.getByText(
-        "profile-promo-email-blocking-description-promotionals-locked-label",
+      within(tooltip).getByText(
```

Comment:
praise, suggestion (non-blocking): TIL about `within`! Very cool function. We should use it more often.